watertap/property_models/seawater_prop_pack.py

```diff
@@ -323,93 +323,142 @@ def build(self):
+            within=Reals,
+            initialize=0.01169,
+            units=enth_mass_units * P_inv_units * t_inv_units,
+            doc="Specific enthalpy parameter A6",
+        )
+        self.enth_mass_param_A7 = Var(
+            within=Reals,
+            initialize=-2.6185e-5,
+            units=enth_mass_units * P_inv_units * t_inv_units**2,
+            doc="Specific enthalpy parameter A7",
+        )
+        self.enth_mass_param_A8 = Var(
+            within=Reals,
+            initialize=7.0661e-8,
+            units=enth_mass_units * P_inv_units * t_inv_units**3,
+            doc="Specific enthalpy parameter A8",
+        )
         self.enth_mass_param_B1 = Var(
             within=Reals,
-            initialize=-2.348e4,
+            initialize=-2.34825e4,
             units=enth_mass_units,
             doc="Specific enthalpy parameter B1",
         )
         self.enth_mass_param_B2 = Var(
             within=Reals,
-            initialize=3.152e5,
+            initialize=3.15183e5,
             units=enth_mass_units,
             doc="Specific enthalpy parameter B2",
         )
         self.enth_mass_param_B3 = Var(
             within=Reals,
-            initialize=2.803e6,
+            initialize=2.80269e6,
             units=enth_mass_units,
             doc="Specific enthalpy parameter B3",
         )
         self.enth_mass_param_B4 = Var(
             within=Reals,
-            initialize=-1.446e7,
+            initialize=-1.44606e7,
             units=enth_mass_units,
             doc="Specific enthalpy parameter B4",
         )
         self.enth_mass_param_B5 = Var(
             within=Reals,
-            initialize=7.826e3,
+            initialize=7.82607e3,
             units=enth_mass_units * t_inv_units,
             doc="Specific enthalpy parameter B5",
         )
         self.enth_mass_param_B6 = Var(
             within=Reals,
-            initialize=-4.417e1,
+            initialize=-4.41733,
             units=enth_mass_units * t_inv_units**2,
             doc="Specific enthalpy parameter B6",
         )
         self.enth_mass_param_B7 = Var(
             within=Reals,
-            initialize=2.139e-1,
+            initialize=2.1394e-1,
             units=enth_mass_units * t_inv_units**3,
             doc="Specific enthalpy parameter B7",
         )
         self.enth_mass_param_B8 = Var(
             within=Reals,
-            initialize=-1.991e4,
+            initialize=-1.99108e4,
             units=enth_mass_units * t_inv_units,
             doc="Specific enthalpy parameter B8",
         )
         self.enth_mass_param_B9 = Var(
             within=Reals,
-            initialize=2.778e4,
+            initialize=2.77846e4,
             units=enth_mass_units * t_inv_units,
             doc="Specific enthalpy parameter B9",
         )
         self.enth_mass_param_B10 = Var(
             within=Reals,
-            initialize=9.728e1,
+            initialize=9.72801,
```

Comment:
Same thing here. Was the previous parameter off by an order of magnitude? Used to be 9.728e1 and now is 9.2780.

Reply:
See above comment